compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -620,7 +611,7 @@ def start(self):
 
     def _event_loop(self):
         parent_identities: set[str] = set()
-        if not is_privileged():
+        if not self.identity_mapper:
```

Comment:
My thinking in using `is_privileged` originally was to "raise the bar."  That is, this is all text, so at some point someone could just modify the if-condition in the code and move on, but by using `is_privileged()` here, that precludes a manual setting of `identity_mapper` after initializing the MEP instance.  Consider:
```python
if not (is_privileged() and self.identity_mapper):
```